spec/models/post_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe "Associations" do
+    it { should belong_to(:user) }
+    it { should have_many(:post_pictures).dependent(:destroy) }
+    it { should have_many(:post_favorites).dependent(:destroy) }
+    it { should have_many(:like_users).through(:post_favorites).source(:user) }
+    it { should belong_to(:ward).optional }
+  end
+
+  describe "Validations" do
+    it { is_expected.to validate_presence_of(:user_id) }
```

Comment:
cho line này -> line 21 vào for each đi e ơi

Reply:
vâng ạ